code/modules/hydroponics/unique_plant_genes.dm

```diff
@@ -446,7 +446,7 @@
 /// Killer Tomato's transformation gene.
 /datum/plant_gene/trait/mob_transformation/tomato
 	dangerous = TRUE
-	killer_plant = /mob/living/simple_animal/hostile/killertomato
+	killer_plant = /mob/living/basic/killer_tomato
```

Comment:
`/datum/plant_gene/trait/mob_transformation/proc/awaken(obj/item/our_plant)` needs to be updated to support this

Reply:
ah thanks for catching that i didn't realize that was a feature my bad